components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/resources/userstore.yaml

```diff
@@ -749,3 +785,35 @@ components:
         value:
           type: string
           example: basic
+    UserStoreAttributeMapping:
+      type: object
+      properties:
+        typeName:
+          type: string
+          example: UniqueIDJDBCUserStoreManager
+        typeId:
+          type: string
+          example: b3JnLndzbzIuY2FyYm9uLnVzZXIuY29yZS5qZGJjLkpEQkNVc2VyU3RvcmVNYW5hZ2Vy
+        isLocal:
+          type: boolean
+          description: Is the userstore is a local userstore or not
+          example: true
+        attributeMappings:
+          type: array
+          items:
+            $ref: '#/components/schemas/UserStoreAttributeDO'
+    UserStoreAttributeDO:
```

Comment:
we normally don't use the DO convention in swagger definitions. Shall we change this to UserStoreAttribute?

Reply:
fixed